morsssss/amp-ecomm
# New template for product-details + new partials

* New template for product-details page, based on a new structure of partials.
* The template includes mustache fields to be server-side rendered (works with dynamic data).
* The directory "partials_new" contains the new partials: head.html, navbar.html, cart-items.html, related_products.html and footer.html.
* The selection of the partials was made based on the need of other pages to reuse these components. 
* Some of the original partials from ampstart.com were not included, for simplicity (ampstart has more partials that are reused across different demos).

To test:

1. Run 'gulp build'. This will generate the files on the /dist directory.
2. Go to dist/ directory to see the output product-details.html.
3. Go to dist/server to see the server.js
4. At dist/server, run 'node server.js' to start the server.
5. Request an example product on a browser window: http://localhost:8080/product-details?productId=91675
